.github/workflows/ci.yml

```diff
@@ -128,7 +128,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - run: docker build --target builder .
+      - name: Set up QEMU
```

Comment:
It would be nice to mention here why we have to use QEMU

Reply:
It's because we want to build cross-platform images, but yes, a comment would be nice.

Reply:
Done.